api/src/pcapi/core/subscription/api.py

```diff
@@ -76,21 +77,22 @@ def activate_beneficiary_for_eligibility(
 
     deposit = finance_api.create_deposit(
         user,
-        deposit_source=deposit_source,
+        deposit_source=fraud_check.get_detailed_source(),
         eligibility=eligibility,
         age_at_registration=age_at_registration,
     )
 
     db.session.add_all((user, deposit))
     db.session.commit()
-    logger.info("Activated beneficiary and created deposit", extra={"user": user.id, "source": deposit_source})
+    logger.info("Activated beneficiary and created deposit", extra={"user": user.id, "source": deposit.source})
 
     is_email_sent = transactional_mails.send_accepted_as_beneficiary_email(user=user)
     if not is_email_sent:
         logger.warning("Could not send accepted as beneficiary email to user", extra={"user": user.id})
 
     external_attributes_api.update_external_user(user)
     batch.track_deposit_activated_event(user.id, deposit)
+    deposit_activation.track_deposit_granted_event(user.id, deposit, fraud_check)
```

Comment:
(détail / question de goût) Autant la ligne du dessus est claire (bon, ok, claire si on sait ce qu'est Batch, évidemment !), autant celle-ci est plus mystérieuse. Suggestion : appeler la fonction comme suit : 

```suggestion
    amplitude_events.track_deposit_granted_event(user.id, deposit, fraud_check)
```

Ainsi, on comprend mieux de quoi ça parle (sans avoir à regarder d'où vient `deposit_activation`.

Ce serait possible en faisant un import de ce genre : `import pcapi.analytics.amplitude.events as amplitude_events`, et en important toutes les fonctions `track_xxxx()` dans le fichier `analytics/amplitude/events/__init__.py`. On fait le même genre de choses pour les mails, cf. `core/mails/transactional/__init__.py`.

Reply:
+1
Le naming devrait ressembler à celui de batch vu que ça fait sensiblement la même chose (envoyer un event à un 3rd party)

```suggestion
    amplitude_events.track_deposit_activated_event(user.id, deposit, fraud_check)
```

Reply:
Bonus : on pourrait avoir une unique fonction qui track les events pour les 3rd parties
Comme ç un seul appel dans `activate_beneficiary_...` et la fonction serait responsable d'envoyer les events aux différents 3rd parties (Batch, amplitude. Pour l'instant on a que ces 2 là)